docs/troubleshooting.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 9
+title: 🛠️ Troubleshooting
+description: Fixes for common Shorebird issues
+---
+
+# 🛠️ Troubleshooting
+
+Find help for issues with Shorebird.
+
+## I installed Shorebird, and now I can't run my app in VS Code
+
+If you see error output like the following when using the Run or Debug button in
```

Comment:
lgtm, we should link to the bug Felix filed though.

Reply:
Done.